.github/workflows/publish.yml

```diff
@@ -32,6 +33,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
   release:
+    permissions:
+      contents: write
```

Comment:
Shouldn't this also say `read` as we also have `actions/checkout` in place? Or is it implied that when you can write you automatically can read as well.

Reply:
It assumes read too, i.e. read-write.